spyder/widgets/sourcecode/base.py

```diff
@@ -293,8 +293,8 @@ def set_tab_stop_width_spaces(self, tab_stop_width_spaces):
         self.update_tab_stop_width_spaces()
 
     def update_tab_stop_width_spaces(self):
-        self.setTabStopWidth(self.fontMetrics().width(
-                '9' * self.tab_stop_width_spaces))
+        self.setTabStopWidth(self.fontMetrics().width(' ')

```

Comment:
@ccordoba12 I changed this `'9'` to a space so If this pops up again it would be obvious. 

After thinking about it however if this code is used with non-monospaced fonts, like in the help widget, then the tabs will be much too small (12px instead of 40px).

It should probably still be a `'9'`

Reply:
This is only used in widgets with mono-spaced fonts. Besides, that `9` has always confused us.

Reply:
I mean, it was put there by Pierre.

Reply:
Ok it will fine as is then. 

Yeah I had to go the Qt documentation on that one.